extensions/ray-so/package.json

```
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ray-so",
+  "title": "Ray.so",
+  "description": "Create beautiful images of your code",
+  "icon": "icon.png",
+  "author": "Garrett",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "create-a-snippet",
+      "title": "Create Beautiful Images of Code",
```

```suggestion
      "title": "Create Image from Code",
```

I would suggest naming for similar [script command](https://github.com/raycast/script-commands/blob/master/commands/developer-utils/create-image-from-code.sh)